src/EntityRepository/CustomerRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\EntityRepository;
+
+use App\Entity\Customer;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+
+/**
+ * @ORM\Entity
+ */
+class CustomerRepository extends ServiceEntityRepository
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function getEntityClass(): string
+    {
+        return Customer::class;
```

Comment:
This could be an issue for some versions of Symfony ORM package with Doctrine, but this needs to be checked.